src/ripple/app/paths/Pathfinder.h

```diff
@@ -176,6 +176,7 @@ class Pathfinder
 
     Account mSrcAccount;
     Account mDstAccount;
+    Account mEffectiveDst;
```

Comment:
This needs documentation!

What's the difference between the "effective destination" and "destination?

What about `mEffectiveDstAccount` to make it more similar to the other variables?